vendor/bat-native-ledger/include/bat/ledger/ledger_callback_handler.h

```diff
@@ -35,9 +35,9 @@ LEDGER_EXPORT enum Result {
   TIP_ERROR = 16,
   CORRUPTED_WALLET = 17,
   GRANT_ALREADY_CLAIMED = 18,
-  // some more useful result codes should go here
 
   LEDGER_RESULT_END = 19,
+  CONTRIBUTION_AMOUNT_TOO_LOW = 20,
```

Comment:
But ledger result end is before the end of the results?

Reply:
this are just enums, we will be adding a lot more of them in here

Reply:
not sure why that one was added `LEDGER_RESULT_END` as we don't use it anywhere

Reply:
Agreed, so as not to confuse maybe best to remove LEDGER_RESULT_END

Reply:
removed